packages/gatsby-source-drupal/src/utils.js

```diff
@@ -468,7 +469,7 @@ export function drupalCreateNodeManifest({
       )
       warnOnceToUpgradeGatsby = true
     }
-    const manifestId = `${id}-${updatedAt}`
+    const manifestId = `${id}-${langcode}-${updatedAt}`
```

Comment:
Will this even match the ID in Drupal of the previous revision since the timestamp should be different and more recent than the last revision? See this line on the Drupal side: https://git.drupalcode.org/issue/gatsby-3277571/-/blob/2.0.x/gatsby.module#L154